index.js

```diff
@@ -1,17 +1,41 @@
+var path = require('path')
 var fs = require('fs-extra')
+var MatcherCollection = require('matcher-collection')
 
 exports.readSync = readSync
-function readSync (dir) {
+function readSync (dir, options, relativeRoot) {
+  options = options || {}
+  var include = options.include
+  var exclude = options.exclude || []
+  relativeRoot = relativeRoot || '';
+
+  var includeMatcher
+  if (include) {
+    includeMatcher = new MatcherCollection(include)
+  }
+  var excludeMatcher = new MatcherCollection(exclude)
```

Comment:
if there is no exclude, should we bother with an excludeMatcher

Reply:
changed to include that optimization